### ISSUE TITLE
[UI] Improve layout of expanded scopes

### ISSUE THREAD

**Issue by helfi92** [taskcluster/taskcluster#1539]
**Describe the bug**

**To Reproduce**
1. Go to https://taskcluster-ui.herokuapp.com/auth/roles/hook-id%3Agarbage%2Fcron-task-johanlorenzo-fenix%2Fnightly
2. Look at the expanded scopes

**Actual behavior**
There is a lot of white space between each scope and the font size is a bit too large.

<img width="799" alt="Screen Shot 2019-10-08 at 5 35 21 PM" src="https://user-images.githubusercontent.com/3766511/66435094-0793e080-e9f2-11e9-9bdd-a783d5a85a1e.png">

**Expected behavior**
There is less white space and the font-size should sit more harmoniously.

**Additional context**
We use material-ui for our components. Refer to https://material-ui.com/ for more information on things like what props are accepted by a component.

**Comment by ajinkabeer** [comment]
I would like to take up this issue if that's possible. 
@helfi92 @djmitche

**Comment by chefoce** [comment]
@helfi92 thanks I would start working right away

**Comment by susanreenesaa** [comment]
if @chefoce  did not take up this task, you can allocate it to me, please

**Comment by chefoce** [comment]
Yes, i'm working, i'm very confused, but i would like to keep triying if its posible



Enviado desde mi Samsung Mobile de Telcel


-------- Mensaje original --------
De: Hassan Ali <notifications@github.com>
Fecha: 10/10/19 9:12 a. m. (GMT-07:00)
A: taskcluster/taskcluster <taskcluster@noreply.github.com>
CC: Carlos Oceguera <chefoceguera@hotmail.com>, Mention <mention@noreply.github.com>
Asunto: Re: [taskcluster/taskcluster] [UI] Improve layout of expanded scopes (#1539)


@chefoce<https://github.com/chefoce> Are you still working on this issue?

**Comment by helfi92** [comment]
@chefoce Are you still working on this issue?

**Comment by km-js** [comment]
Hey @helfi92 . I would like to work on here, if @chefoce is not available.

**Comment by JiwoonKim** [comment]
@helfi92 I would like to work on this issue, if @km-js is not available.

**Comment by helfi92** [comment]
How is it going @JiwoonKim? :-)

**Comment by km-js** [comment]
Should I try??
